gateway/engine/beans/pom.xml

```diff
@@ -12,6 +12,10 @@
 
   <dependencies>
     <dependency>
+      <groupId>net.openhft</groupId>
+      <artifactId>zero-allocation-hashing</artifactId>
+    </dependency>
+    <dependency>
```

Comment:
Might as well ask here. @rnc mind checking whether this dep is OK and if not imported already if that's doable (there are [no transitive dependencies](https://github.com/OpenHFT/Zero-Allocation-Hashing/blob/master/pom.xml) so it should be easy peasy).